lib/views/pages/profile/profile_page.dart

```diff
@@ -1,79 +1,108 @@
 import "package:flutter/material.dart";
 import "package:hooks_riverpod/hooks_riverpod.dart";
 import "package:proxima/viewmodels/profile_view_model.dart";
-import "package:proxima/views/pages/profile/posts_info/info_card.dart";
+import "package:proxima/views/pages/profile/posts_info/info_card_badge.dart";
+import "package:proxima/views/pages/profile/posts_info/info_card_comment.dart";
+import "package:proxima/views/pages/profile/posts_info/info_card_post.dart";
 import "package:proxima/views/pages/profile/posts_info/info_column.dart";
 import "package:proxima/views/pages/profile/posts_info/info_row.dart";
-import "package:proxima/views/pages/profile/user_info/centauri_points.dart";
 import "package:proxima/views/pages/profile/user_info/user_account.dart";
 
 /// This widget is used to display the profile page
 /// It contains the user info, centauri points, badges, posts and comments
 class ProfilePage extends HookConsumerWidget {
   const ProfilePage({super.key});
+  static const postTabKey = Key("post tab");
+  static const commentTabKey = Key("comment tab");
+  static const tabKey = Key("tab");
+  static const postColumnKey = Key("post column");
+  static const commentColumnKey = Key("comment column");
```

Comment:
Just applying camelCase :+1: 
```suggestion
  static const postTabKey = Key("postTab");
  static const commentTabKey = Key("commentTab");
  static const tabKey = Key("tab");
  static const postColumnKey = Key("postColumn");
  static const commentColumnKey = Key("commentColumn");
```